eodag/plugins/search/creodias_s3.py

```diff
@@ -117,7 +117,14 @@ def _update_assets(product: EOProduct, config: PluginConfig, auth: AwsAuth):
 
 class CreodiasS3Search(ODataV4Search):
     """
-    Search on creodias and adapt results to s3
+    CreodiasS3Search is an extension of ODataV4Search, it executes a Search on creodias and
+    adapts results so that the assets contain links to s3. It has the same configuration
+    parameters as ODataV4Search and one additional parameter:
+
+    :param provider: provider name
+    :param config: Search plugin configuration:
+
+        * :attr:`~eodag.config.PluginConfig.s3_endpoint` (``str``) (**mandatory**): base url of the s3
```

Comment:
s3_endpoint actually does exist in the PluginConfig but as a parameter of the AwsDownload config, maybe we should rename one of the two to avoid confusion.